src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,44 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 20px;
+
+  @include onTablet {
+    grid-template-columns: repeat(6, 1fr);
+    column-gap: 24px;
+  }
+
+  @include onDesktop {
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  &--params {
+    display: block;
```

Comment:
It's a common mistake to do such a way.
Use `grid-template-columns: 1fr;` instead.